web/webkit/src/main/scala/net/liftweb/http/js/JsCommands.scala

```diff
@@ -793,12 +793,12 @@ object JsCmds {
   implicit def jsExpToJsCmd(in: JsExp) = in.cmd
 
   case class CmdPair(left: JsCmd, right: JsCmd) extends JsCmd {
-    import scala.collection.mutable.ListBuffer;
+    import scala.collection.mutable.ListBuffer
 
     def toJsCmd: String = {
       val acc = new ListBuffer[JsCmd]()
       appendDo(acc, left :: right :: Nil)
-      acc.map(_.toJsCmd).mkString("\n")
+      acc.map(_.toJsCmd).filterNot(_.isEmpty).mkString("\n")
```

Comment:
Actually, rather than do this, why not change `appendDo` to simply never append Noops? It avoids an extra pass through the list if nothing else, though I doubt there are typically large enough JsCmd combinations for that to make a big difference.